grass7/raster/r.in.srtm.region/r.in.srtm.region.py

```diff
@@ -192,22 +215,57 @@ def download_tile(tile, url, pid, srtmv3, one, username, password):
         except:
             goturl = 0
             pass
-        
+
         if goturl == 1:
             return 1
 
     return 0
-    
-    
+
+
 def cleanup():
     if not in_temp:
         return
     os.chdir(currdir)
     grass.run_command('g.region', region = tmpregionname)
     grass.run_command('g.remove', type = 'region', name = tmpregionname, flags = 'f', quiet = True)
     grass.try_rmdir(tmpdir)
+    if TGTGISRC:
+        os.environ['GISRC'] = str(TGTGISRC)
+    # remove temp location
+    if TMPLOC:
+        grass.try_rmdir(os.path.join(GISDBASE, TMPLOC))
+    if SRCGISRC:
+        grass.try_remove(SRCGISRC)
+
+
+def createTMPlocation(epsg=4326):
+    SRCGISRC = grass.tempfile()
+    TMPLOC = 'temp_import_location_' + str(os.getpid())
+    f = open(SRCGISRC, 'w')
+    f.write('MAPSET: PERMANENT\n')
+    f.write('GISDBASE: %s\n' % GISDBASE)
+    f.write('LOCATION_NAME: %s\n' % TMPLOC)
+    f.write('GUI: text\n')
+    f.close()
+
+    # create temp location from input without import
+    grass.verbose(_("Creating temporary location with EPSG:%d...") % epsg)
+
+    cmd = grass.Popen("grass78 -c epsg:%d -e %s" % (epsg, os.path.join(GISDBASE, TMPLOC)),
```

Comment:
... this should be replaced with `g.proj` code (as an example: `g.proj -c epsg=4326 location=latlong`)

Reply:
Is changed